bugbug/repository.py

```diff
@@ -50,6 +53,48 @@
 files_touched_prev = defaultdict(int)
 files_touched_prev_90_days = defaultdict(int)
 
+# This is only a temporary hack: Should be removed after the template issue with reviewers
+# gets fixed. Most of this code is copied from https://github.com/mozilla/version-control-tools/blob/2c2812d4a41b690203672a183b1dd85ca8b39e01/pylib/mozautomation/mozautomation/commitparser.py#L129
```

Comment:
Add a link to the bug I mentioned `gets fixed (BUGZILLA LINK).`

Reply:
Can you move the comment right on top of the function, without the two newlines?